geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -187,27 +193,49 @@ The header can be in a separate file, diff in the `*.html.ejs` file:
 
 The content will be in the `geoportal/{{package}}_geoportal/static/header.html` file.
 
+Print
+-----
 
-Information
+- With the default configuration we can easier switch from local print and mutualised print
+- A new attribute is added and containy the client timezone
```

Comment:
Suggestion:
"The client timezone is added as a new attribute"

(but it would be interesting to know what happens with this new attribute?)

Reply:
I will add an example :-)